README.md

```diff
@@ -133,7 +141,7 @@ Quick legend:
 | Data Generation                                                       |                                 :construction:                                 |         ✔         |
 | Loot Table API                                                        |                                       ❌                                        |         ✔         |
 | Resource Conditions                                                   |                                       ❌                                        |         ✔         |
-| Component API (like CCA or Forge capabilities)                        | [:construction:](https://github.com/QuiltMC/quilt-standard-libraries/pull/146) |         ❌         |
+| Component API (like CCA or Forge capabilities)                        | [:construction:](https://github.com/QuiltMC/quilt-standard-libraries/pull/146) | ✔ (with Data Attachment API) |
```

Comment:
I think making some way to indicate a "stalled" PR/module. Makes it easy for people to know something they should take over

Reply:
What about 🧱? "bricked software"

Reply:
I feel like bricked and stalled are different

Reply:
What about ⏳? I don't have any other ideas

Reply:
Thats not bad, lets go with that for now!